### PR TITLE
Add cli image

### PR DESCRIPTION
**Pull request by jamilraichouni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -304,7 +304,26 @@ docker build -t $BASE/ease \
...
+executable.
+
+You can build a CLI image with the following command,
+whereby you replace `$BASE` with `capella` or `t4c/client`.
+
+```zsh
+docker build -t $BASE/cli --build-arg BASE_IMAGE=$BASE/base cli
+```
+
+Please replace $BASE with `capella` or `t4c/client`.
```
Comment (MoritzWeber0) [inline README.md:324]: This was already described in lines 317-318. I'd remove it.

File: README.md [hunk 2]
```diff
@@ -419,6 +438,40 @@ docker run -v script.py:/opt/scripts/script.py $BASE/ease
 
 where `$BASE` is again `capella` or `t4c/client`.
 
+### CLI container
+
+```zsh
+docker run $BASE/cli -nosplash -consolelog -application APPLICATION -appid APPID [...]
```
Comment (MoritzWeber0) [inline README.md:444]: Doesn't it make sense to include `-nosplash -consolelog` in the entrypoint directly? As far as I know, it makes sense for all of our applications.

**Review by MoritzWeber0 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (jamilraichouni) [reply README.md:442]: Initially I had it exactly as you propose and decided against it.

Reasons:

(A)
I want the command calling Capella (local exe or Docker image) to be interchangeable as much as possible.

Example (pseudo-code):

```bash
export CAPELLA_VERSION=5.2
CAPELLA_CMD="docker run --rm -it capella/cli:$CAPELLA_VERSION"
$CAPELLA_CMD -nosplash -consolelog -application APP -appid APPID
```

versus

```bash
export CAPELLA_VERSION=5.2
CAPELLA_CMD=/Applications/Capella_5.2.app/Contents/MacOS/capella
$CAPELLA_CMD -nosplash -consolelog -application APP -appid APPID
```
(B)
Already passing these flags would be some kind of an unexpcted and magic internal/ hidden behaviour

If we pass `nosplash` or not does not play a notable role with our virtual display. It works in both cases. And for `-consolelog` I want to give the end user the control to decide.

Comment (MoritzWeber0) [reply README.md:442]: Makes sense 👍

**Pull request merged**